client/landing/stepper/declarative-flow/internals/steps-repository/use-my-domain/index.tsx

```diff
@@ -66,6 +72,7 @@ const UseMyDomain: StepType< {
 		setHideFreePlan( true );
 		setDomainCartItem( domainCartItem );
```

Comment:
I'm thinking of moving that outside of this step as well. In my opinion, the job of adding the item to the cart is from the flow not the step because each flow might do it differently eg modify metadata. That's how unified domains do it. But let me know if we want to do that within the step, so I can change how the domains step works as well @alshakero.

Reply:
Let's only change `use-my-domain` then. It'll be used in several flows (all that use the new `domain-search` step.)